### DISCUSSION TITLE
Shortcuts

### DISCUSSION THREAD

**Discussion by crazzmc** [thebaselab/codeapp#963] [General]
Are there any default shortcuts? I've been looking for documentation, but cannot seem to find anything. Also do you think it would be possible to pre-populate all of the 'custom shortcuts' with an initial value? There are so many, it would be easier to learn/change rather than populate each individual one.

My main question comes from the CMD+P function in VSC I cannot seem to find a shortcut to quick open files. Am I just overlooking something or is that default shortcut to quickly search and open a specific file not implemented/available.

Thanks!

**Comment by bummoblizard** [comment]
The default shortcuts can be viewed by long-pressing command. For the
editor shortcuts, they are the same as VS Code’s by default.

Quick opening is not yet implemented. We will be working on it in the
future.
